cmd/krew/cmd/internal/setup_check_test.go

```diff
@@ -43,6 +43,40 @@ func TestIsBinDirInPATH_secondRun(t *testing.T) {
 	}
 }
 
+// Test case when PATH content is not well normalized
+func TestIsBinDirInPATH_NonNormalized(t *testing.T) {
+	tempDir := testutil.NewTempDir(t)
+
+	// backup and restore initial PATH value
+	backupPath := os.Getenv("PATH")
```

Comment:
Since you are not capturing the variable, but passing as argument, consider to inline in line 57.